internal/rollup/rollup.config.js

```diff
@@ -53,12 +52,14 @@ class NormalizePaths {
 }
 
 export default {
-  output: {format: 'iife'},
+  input: [TMPL_inputs],
+  output: {dir: path.join(process.cwd(), binDirPath, buildFileDirname, 'bundles.es6'), format: 'cjs'},
+  experimentalCodeSplitting: true,
```

Comment:
not that we're really rolling it out yet - but I think we should consider exposing an experimental flag as a rule attribute when we enable these things - we could have users depend on us to provide a higher guarantee than the thing we depend on